### ISSUE TITLE
featureRequest: Daemon Mode

### ISSUE THREAD

**Issue by GNUuser** [geobra/harbour-shmoose#52]
Demon mode helps to notify messages in real time, and keeps the system interface clean.
apps like Conversations.im already use daemon mode. It would be great to integrate this feature into Shmoose.

**Comment by geobra** [comment]
Indeed. It's on my todo list. But currently not with a high priority.
